lib/cookie/cookie.ts

```diff
@@ -421,16 +420,12 @@ export class Cookie {
   sameSite: string | undefined
 
   constructor(options: CreateCookieOptions = {}) {
-    // supports inspect if that feature is available in the environment
-    const customInspectSymbol = getCustomInspectSymbol()
-    if (customInspectSymbol) {
-      Object.defineProperty(this, customInspectSymbol, {
-        value: this.inspect.bind(this),
-        enumerable: false,
-        writable: false,
-        configurable: false,
-      })
-    }
+    Object.defineProperty(this, NODEJS_UTIL_INSPECT_CUSTOM, {
```

Comment:
One change that we could do, but I wanted to discuss before deciding, is whether to move this to be a regular prop on the prototype, rather than a weird bound prop on the instance. Technically a breaking change, but a small one, and we're gonna land a major anyway, so we could get away with it.